### PR TITLE
Initial code

### PR DESCRIPTION
**Pull request by KatKiker**
Transfer of impact study package to new structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+```
+
+However, using PDM is recommended to ensure consistent and reproducible environments.
+
+### Download Ephemeris Files
+
+To download the assist ephemeris files, run:
+
+```
+python set_up_assist.py
```
Comment (akoumjian) [inline README.md:31]: To run this command with the virtualenv python, it looks like you need to activate it. If users are only using pdm, you will need to do something like `eval $(pdm venv activate)`

File: src/adam_impact_study/analysis.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    """
+    for obj in ip_dict.keys():
+        obj_dict = ip_dict[obj]
+        ip_df = pd.DataFrame(list(obj_dict.items()), columns=["Day", "IP"])
+        plt.scatter(ip_df["Day"], ip_df["IP"])
+        plt.title(obj)
+        plt.xlabel("Day")
+        plt.ylabel("Impact Probability")
+        plt.plot(ip_df["Day"], ip_df["IP"])
+        plt.show()
```
Comment (akoumjian) [inline src/adam_impact_study/analysis.py:28]: I would change this to save the plots to an output folder rather than using `.show`.

**Review by akoumjian — APPROVED** [review]
Couple of small changes requested.

**Comment by KatKiker** [comment]
> Couple of small changes requested.

Changed made. Merging.

**Pull request merged**